### DISCUSSION TITLE
The "Current Import" variable at "N" phase (Injet Sonic charger)

### DISCUSSION THREAD

**Discussion by f3peter** [lbbrhzn/ocpp#1310] [Q&A] [answered]
Hello, 

My charger at charging send the following OCPP message: 
`2024-09-05 23:41:59.968 INFO (MainThread) [ocpp] 72380662347719: receive message [2,"121416101","MeterValues",{"connectorId":1,"transactionId":1725564597,"meterValue":[{"timestamp":"2024-09-05T21:41:59.333Z","sampledValue":[{"measurand":"Energy.Active.Import.Register","unit":"Wh","value":"254780"},{"measurand":"Power.Active.Import","unit":"kW","value":"3.5"},{"measurand":"Temperature","unit":"Celsius","value":"35"},{"measurand":"Voltage","phase":"L1-N","unit":"V","value":"231.1"},{"measurand":"Current.Import","phase":"N","unit":"A","value":"15.6"}]}]}]
`
All value are good, and i see them in Home assistant except the Current.Import. The Current Import value still 0. My guess is the phase name is wrong. 
How can I modify the OCPP server to output the current of the "N" phase (instead of the current of the "L1" phase)? 

THX

**Comment by drc38** [comment, marked as answer]
See #1311
